lte/gateway/c/core/oai/lib/s8_proxy/s8_client_api.cpp

```diff
@@ -172,31 +172,37 @@ static void recv_s8_create_session_response(
     OAILOG_ERROR_UE(
         LOG_SGW_S8, imsi64,
         "Failed to allocate memory for S8_CREATE_SESSION_RSP for "
-        "context_teid" TEID_FMT "\n",
-        context_teid);
+        "temporary_create_session_procedure_id:%u sgw_s8_cp_teid " TEID_FMT
+        "\n",
```

Comment:
no need for "\n" as the macro itself takes care of it.

Reply:
Thank you for letting me know. Earlier we explicitly needed to have "\n".